src/import-checks/import-checks.service.ts

```diff
@@ -39,6 +39,14 @@ export class ImportChecksService {
   public async runImportChecks(monPlan: UpdateMonitorPlanDTO) {
     let errorList = [];
 
+    // Version Check
+    if (!monPlan.version) {
+      errorList.push(
+        "JSON Schema is missing the required version attribute.",
+      )
+    };
+    this.checkIfThrows(errorList);
+
```

Comment:
The `@IsNotEmpty` decorator utilized in the suggestion above removes the need for this check here.
```suggestion
```